kong/tracing/propagation.lua

```diff
@@ -14,7 +15,7 @@ local baggage_mt = {
 }
 
 local B3_SINGLE_PATTERN =
-  "^(%x+)%-(%x%x%x%x%x%x%x%x%x%x%x%x%x%x%x%x)%-?([01d]?)%-?(%x*)$"
+"^(%x+)%-(%x%x%x%x%x%x%x%x%x%x%x%x%x%x%x%x)%-?([01d]?)%-?(%x*)$"
```

Comment:
The indent change seems not to be needed.

Reply:
fixed